### PR TITLE
Firefox 122 release notes for WebDriver conformance

### PR DESCRIPTION
**Pull request by whimboo**
Here the [list of fixed bugs](https://bugzilla.mozilla.org/buglist.cgi?v1=fixed&resolution=FIXED&f2=cf_status_firefox121&query_format=advanced&o1=equals&f1=cf_status_firefox122&o2=notequals&status_whiteboard=%5Bwebdriver%3Arelnote%5D&columnlist=component%2Cresolution%2Cassigned_to%2Cshort_desc%2Cbug_type%2Cstatus_whiteboard&status_whiteboard_type=substring&v2=fixed&list_id=16864857) worth a release note on MDN.

CC @lutien and @juliandescottes for review.

### CODE REVIEW

**Review by juliandescottes — COMMENTED** [review]
lgtm

File: files/en-us/mozilla/firefox/releases/122/index.md [hunk 1]
```diff
@@ -67,10 +67,22 @@ This article provides information about the changes in Firefox 122 that affect d
 
 ### WebDriver conformance (WebDriver BiDi, Marionette)
 
+#### General
+
+- Fixed a bug that prevented [Perform Actions](https://w3c.github.io/webdriver/#perform-actions) to correctly synthesize double and other multi-click events for the `mouse` input source ([Firefox bug 1864614](https://bugzil.la/1864614)). Additionally, these events will only be emitted when the actual mouse position has changed since the last click action ([Firefox bug 1681076](https://bugzil.la/1681076)).
```
Comment (lutien) [inline files/en-us/mozilla/firefox/releases/122/index.md:72]: The last sentence should actually say that events are emitted when mouse position has not changed. Or you wanted to say that we fixed this behavior?

Comment (whimboo) [reply files/en-us/mozilla/firefox/releases/122/index.md:72]: Good find. I indeed missed the `not`.

**Comment by whimboo** [comment]
@pepelsbey or @bsmth could you please do the final review? Thanks.

**Review by bsmth — APPROVED** [review]
+1 from me, too. Thank you, all - feel free to merge when ready.

**Comment by whimboo** [comment]
> +1 from me, too. Thank you, all - feel free to merge when ready.

I do not have merge permissions. So could you please do me the favor and merge? Thanks!

**Pull request merged**